compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmObservabilityGen.java

```diff
@@ -110,7 +110,7 @@
     private static final String INVOCATION_INSTRUMENTATION_TYPE = "invocation";
     private static final String FUNC_BODY_INSTRUMENTATION_TYPE = "funcBody";
     private static final Location COMPILE_TIME_CONST_POS =
-            new BLangDiagnosticLocation(null, -1, -1, -1, -1);
+            new BLangDiagnosticLocation(null, -1, -1, -1, -1, 0, 0);
```

Comment:
Why we need this change?

Reply:
The later one is the new API and the previous one has been deprecated.